# Translations template for vigilo-connector-metro.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the vigilo-connector-metro
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-connector-metro 3.5\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2014-09-26 16:39+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/connector_metro/__init__.py:45
msgid ""
"Please set the path to the configuration database generated by VigiConf in "
"the settings.ini."
msgstr ""

#: src/vigilo/connector_metro/bustorrdtool.py:89
#, python-format
msgid "'%(msgtype)s' is not a valid message type for metrology"
msgstr ""

#: src/vigilo/connector_metro/bustorrdtool.py:97
#, python-format
msgid "Not a valid performance message (missing '%(tag)s' tag)"
msgstr ""

#: src/vigilo/connector_metro/bustorrdtool.py:110
#, python-format
msgid "Invalid metrology value: %s"
msgstr ""

#: src/vigilo/connector_metro/bustorrdtool.py:117
#, python-format
msgid "Skipping perf update for host %s"
msgstr ""

#: src/vigilo/connector_metro/bustorrdtool.py:159
#, python-format
msgid "RRDtool could not update the file %(filename)s. Message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:74
#, python-format
msgid "Error in rrdtool output: %s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:157
#, python-format
msgid "Host '%(host)s' with datasource '%(ds)s' not found in the configuration"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:178
#, python-format
msgid "RRDtool could not create the file: %(filename)s. Message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:284
#, python-format
msgid "Unable to create the directory '%s'"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:321
#, python-format
msgid "Unable to start \"%(rrdtool)s\". Make sure the path is correct."
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:324
#, python-format
msgid ""
"Unable to start \"%(rrdtool)s\". Make sure RRDtool is installed and you have "
"permissions to use it."
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:338
#, python-format
msgid "The RRD directory does not exist: %s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:342
#, python-format
msgid "Unable to write in the directory '%(dir)s'"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:401
#, python-format
msgid "Started RRDtool subprocess: pid %(pid)d"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:423
msgid "The process has not yet completed the previous job"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:461
msgid "No deferred available in _handle_result(), this should not happen"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:488
msgid "The RRDTool process exited normally"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:491
#, python-format
msgid ""
"The RRDTool process was terminated abnormally with exit code %(rcode)s and "
"message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:501
msgid "Restarting..."
msgstr ""

